src/Evolution/EventsAndDenseTriggers/DenseTrigger.hpp

```diff
@@ -79,7 +81,7 @@ class DenseTrigger : public PUP::able {
                   },
                   box);
           if (result.is_triggered) {
-            previous_trigger_time_ = db::get<::Tags::Time>(box);
+            next_previous_trigger_time_ = db::get<::Tags::Time>(box);
```

Comment:
This changes the behavior of `previous_trigger_time()` so it does not match its documentation anymore.  Change one or the other.